packages/lambdoc/lambdoc.1.0-beta3/opam

```diff
@@ -1,31 +1,31 @@
-opam-version: "1"
-maintainer: "dario.teixeira@nleyten.com"
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
 authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
-homepage: "http://lambdoc.forge.ocamlcore.org/"
+homepage: "https://lambdoc.forge.ocamlcore.org/"
```

Comment:
This URL gives me a TLS error because the cert is only for forge.ocamlcore.org and ocamlcore.org.